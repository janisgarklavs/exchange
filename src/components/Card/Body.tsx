import React from "react";

const styleCX: React.CSSProperties = {};

export const Body: React.SFC = ({ children }) => (
  <div style={styleCX}>{children}</div>
);
