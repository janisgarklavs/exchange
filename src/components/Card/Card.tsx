import React from "react";

const styleCX: React.CSSProperties = {
  maxWidth: 400,
  margin: "0 auto",
  padding: 24,
  boxShadow: "0 0 5px rgba(0, 0, 0, .2)",
  background: "white"
};

export const Card: React.SFC = ({ children }) => (
  <div style={styleCX}>{children}</div>
);
