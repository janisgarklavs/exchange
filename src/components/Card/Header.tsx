import React from "react";

const styleCX: React.CSSProperties = {
  display: "flex",
  flexDirection: "column",
  justifyContent: "center"
};

export const Header: React.SFC = ({ children }) => (
  <div style={styleCX}>{children}</div>
);
