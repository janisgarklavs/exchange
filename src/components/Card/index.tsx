export { Card } from "./Card";
export { Header as CardHeader } from "./Header";
export { Body as CardBody } from "./Body";
