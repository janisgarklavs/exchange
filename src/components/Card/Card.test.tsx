import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";

import { Card } from "./Card";

describe("<Card />", () => {
  test("it renders correctly", () => {
    const component = shallow(<Card />);

    expect(toJson(component)).toMatchSnapshot();
  });
});
