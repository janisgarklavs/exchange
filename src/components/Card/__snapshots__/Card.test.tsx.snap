// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Card /> it renders correctly 1`] = `
<div
  style={
    Object {
      "background": "white",
      "boxShadow": "0 0 5px rgba(0, 0, 0, .2)",
      "margin": "0 auto",
      "maxWidth": 400,
      "padding": 24,
    }
  }
/>
`;
